src/ocean/text/json/Json.d

```diff
@@ -31,60 +31,61 @@ import ocean.text.json.JsonParser;
 
 import Float = ocean.text.convert.Float;
 
-version(UnitTest) import ocean.core.Test;
-
-/*******************************************************************************
+version (UnitTest)
+{
+    import ocean.core.Test;
+    import ocean.io.Stdout;
+}
 
-    Parse json text into a set of inter-related structures. Typical
-    usage is as follows:
-    ---
-    auto json = new Json!(char);
-    json.parse (`{"t": true, "n":null, "array":["world", [4, 5]]}`);
-    ---
-
-    Converting back to text format employs a delegate. This one emits
-    document content to the console:
-    ---
-    json.print ((char[] s) {Stdout(s);});
-    ---
-
-    Constructing json within your code leverages a handful of factories
-    within a document instance. This example creates a document from an
-    array of values:
-    ---
+///
+unittest
+{
+    /*
+        Typical usage is as follows:
+    */
```

Comment:
I think our normal style for both single and multi-line inline documentation is `//`. Same goes for the other comments.

Reply:
There you go!